src/chart/Treemap.tsx

```diff
@@ -18,7 +18,6 @@ import { Global } from '../util/Global';
 import { findChildByType, validateWidthHeight, filterProps } from '../util/ReactUtils';
 import { AnimationDuration, AnimationTiming, DataKey, Margin } from '../util/types';
 import { ReportChartMargin, ReportChartSize } from '../context/chartLayoutContext';
-import { TooltipContextValue } from '../context/tooltipContext';
```

Comment:
removing dead code in Treemap

Reply:
It's easier to review stuff when changes like this are isolated 😬